### PR TITLE
Portfolio: add a contact form and finish the mobile version.

### PR DESCRIPTION
**Pull request by shm04**
### In this pull request we:

- Add contact form and footer in index.html
- Add contact form and footer style in style.css

### CODE REVIEW

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @shm04 @ghreza-crypto 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Descriptive pull request summary ✔️ 
- Good commit messages ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 1]
```diff
@@ -121,5 +121,27 @@ <h1 class="about-title">About</h1>
...
     </section>
+    <section class="footer get-started">
+        <h1 class="title-get-s">Get started</h1>
+        <p class="text-get-s">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+        <form class="search-boxes" action="https://formspree.io/f/xvonzpwb" method="post"> 
+            <input class="name-email search1" type="text" placeholder="Your name" required maxlength="30">
+            <input class="name-email search2" type="email" placeholder="Your email" required>
+            <textarea class="message search3" placeholder="Enter yout message here..." required maxlength="500"></textarea>
+            <button type="submit" class="start-btn bottom">Start collaboration</button>
+        </form>
```
Comment (brainconnect93) [inline index.html:132]: - [ ] Having made a great effort working on the changes requested by the previous reviewer, I noticed the form submission is still not working, anytime I tried to submit a message via the form it throws an error.
  - I would kindly suggest you make the `required` attribute come last in the `input` tag i.e it should come after the max length attribute.
  - Also give the input tag a `name attribute` with the value of `fullName`. I believe with these changes the error will be resolved and I would advise you to try and submit a message via the form from your end and be sure the form is submitted successfully before submitting it for another review.

Error message 👇 

form1

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @shm04, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Tadesse-Alemayehu — APPROVED** [review]
### Hi Team 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**